uktrade/data-workspace-frontend
# feat/DT-1990: Inline feedback

### Description of change

Ticket - [DT-1990](https://uktrade.atlassian.net/jira/software/projects/DT/boards/356?selectedIssue=DT-1990)

This change adds the Inline feedback component #3186 to the data catalogue page as we want to get data around how helpful this page is for the user.

### How to test
1. Visit `/datasets/`
2. Scroll down to the inline feedback component. You should see a question: "Was it helpful?"
3. Click one of the buttons (Yes or No) - once clicked this gets recorded into the database
4. After submitting Yes or No you should then see an additional form asking for more details
5. Fill out the additional form and click "Send feedback" 
6. After submission the additional form should disappear leaving the message: "Thanks for letting us know, your response has been recorded."

### Checklist


[DT-1990]: https://uktrade.atlassian.net/browse/DT-1990?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
